data/os/windows.yaml

```diff
@@ -1,12 +1,12 @@
 ---
 facter::facts_d_dir: C:\ProgramData\PuppetLabs\facter\facts.d
-facter::facts_d_owner: BUILTIN\Administrators
-facter::facts_d_group: BUILTIN\Administrators
+facter::facts_d_owner: NT AUTHORITY\SYSTEM
+facter::facts_d_group: NT AUTHORITY\SYSTEM
```

Comment:
does this differ by version of windows?

Reply:
I have no idea :).  I have only tested this on Windows 2012 R2 and Windows 2016 and the default was `NT AUTHORITY\SYSTEM`.  A system with Puppet 5.3.3 already installed did not change ownership with value set to `NT AUTHORITY\SYSTEM`.

I did observe that 2012 R2 changed from `BUILTIN\Administrators` to `NT AUTHORITY\SYSTEM` for the directory created by the Exec.  The other directories and files were not changed.  My guess is this has to do with the user context executing the Exec in windows.